platform/openide.awt/src/org/openide/awt/QuickSearch.java

```diff
@@ -354,6 +359,23 @@ public void keyTyped(KeyEvent e) {
         if(isAlwaysShown()) {
             displaySearchField();
         }
+        component.getActionMap().put("openQuickSearch", new AbstractAction() {
+          @Override
+          public void actionPerformed(ActionEvent evt) {
+            if (searchPanel != null && isEnabled()) {
+                searchTextField.selectAll();
+                return;
+            }
+            if (searchPanel != null || !isEnabled()) {
+                return;
+            }
+            searchTextField.setText("");
+            displaySearchField();
+          }
+        });
+        InputMap im = component.getInputMap(JComponent.WHEN_ANCESTOR_OF_FOCUSED_COMPONENT);
+        im.put(Utilities.stringToKey("D-F"), "openQuickSearch");
```

Comment:
This is a problem in Project and Files views in the IDE as it overrides the project wide searches.